### PR TITLE
Vimeo.js -- fix eslint warnings

### PR DESCRIPTION
**Pull request by retorquere**

### CODE REVIEW

**Comment by retorquere** [comment]
updated for @zotero/eslint-config@1.0.2

File: Vimeo.js [hunk 1]
```diff
@@ -182,5 +184,5 @@ var testCases = [
 		"url": "https://vimeo.com/search?q=zotero",
 		"items": "multiple"
 	}
-]
-/** END TEST CASES **/
\ No newline at end of file
+];
```
Comment (zuphilip) [inline Vimeo.js]: no

File: Vimeo.js [hunk 2]
```diff
@@ -82,26 +84,26 @@ function getSearchResults(doc, checkOnly) {
...
-	} else {
+	}
+	else {
 		scrape(doc, url);
 	}
 }
 
 
-function scrape(doc, url) {
+function scrape(doc) {
```
Comment (zuphilip) [inline Vimeo.js]: `_url`

**Comment by zuphilip** [comment]
Thank you!

**Pull request merged**